src/main/kotlin/pl/put/smartgarden/infra/configuration/LocalizationConfiguration.kt

```diff
@@ -0,0 +1,30 @@
+package pl.put.smartgarden.infra.configuration
+
+import org.springframework.context.MessageSource
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.support.ReloadableResourceBundleMessageSource
+import org.springframework.validation.beanvalidation.LocalValidatorFactoryBean
+import java.util.*
+
+
+@Configuration
+class LocalizationConfiguration {
```

Comment:
o kurde a nie mieliśmy tego podawać z łapy?

Reply:
miała chyba być fancy mapka

Reply:
Localization odnosi się do różnych wersji językowych, bo początkowo chciałem zwracać informacje po polsku i po angielsku. Właściwie to jest to pod to przygotowane (pliki messages.properties), ale to już od frontu zależy jak to będzie wyglądało :D Na chwilę obecna tylko walidacja wspiera wiadomości w różnych językach - domyślnie angielski, ale gdyby ktoś w headerze podał Accept-Language to zwrócone informacje wrócą po polsku :)